app/PimplePSRContainerBridge.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Silex;
+
+use InvalidArgumentException;
+use Pimple;
+use Psr\Container\ContainerInterface;
+
+final class PimplePSRContainerBridge implements ContainerInterface
```

Comment:
I think we might also be able to use this as a temporary solution while migrating to an actual PSR solution. But to be investigated further when we get to that point.